flink-core/src/main/java/org/apache/flink/core/execution/JobClient.java

```diff
@@ -35,6 +35,11 @@
 @PublicEvolving
 public interface JobClient extends AutoCloseable {
 
+	@Override
+	default void close() {
```

Comment:
@aljoscha this is the commit we don't throw checked exception in `JobClient#close`. The idea is that we cannot do something for it so that it is unreasonable to be checked.

Reply:
Cool, thanks for the info! 🍜I can then change this in my PR.